### PR TITLE
feat(signup): add user

### PR DESCRIPTION
**Pull request by Patience250**
### What does this PR do?
creating user account
### Description of tasks to be completed
- Create route for registering new user
- Add User model
- Add user controller 
### How to test manually
- Clone the repository or pull from remote and change to "ft-signup" branch
- run `npm install && npm run db:init` 
- run `npm start` and open in your postman `http://localhost:4000/api/users/register`

### CODE REVIEW

**Review by RedJanvier — APPROVED** [review]
LGTM

**Review by franko-bit — APPROVED** [review]
Great work Buro

File: src/controller/users.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+const db = require("../database/models/index")
+const User = db.User
+let newUser;
+// creating an User 
+exports.createUser = async(req, res) => {
+    let { email, password, confirm_password } = req.body
```
Comment (niyodusengaclement) [inline src/controller/users.js]: U should catch all errors, u can use premises or try and catch

File: src/controller/users.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    if (!email) return res.status(400).send({ error: "Please provide a email" })
```
Comment (niyodusengaclement) [inline src/controller/users.js]: Use email pattern or use joi bcz if the user type anything here system will let him in which is not good

**Review by niyodusengaclement — CHANGES_REQUESTED** [review]

**Review by Siyubu — APPROVED** [review]

**Review by niyodusengaclement — APPROVED** [review]

**Pull request merged**